### ISSUE TITLE
Linux: Ubuntu / Kubuntu missing libpng12.so.0

### ISSUE THREAD

**Issue by Zyklame** [citra-emu/citra#2564]
I'am using Kubuntu 16.10 (YakketyYak)
Since these version the library libpng12.so.0 is not available anymore in the repository to download and was replaced by libpng16.so.16
With these citra cannot be started at all.
Error:
"./citra-qt: error while loading shared libraries: libpng12.so.0: cannot open shared object file: No such file or directory"

Tested with: 	Nightly Build - e10b11a

**Comment by j-selby** [comment]
Yep - recompile from source if you can.

**Comment by Zyklame** [comment]
Compiling Citra myself is working for now until the build server is updated.

**Comment by jfmherokiller** [comment]
this sounds like an issue with travis-ci

**Comment by j-selby** [comment]
The problem here is that <=16.04 uses libpng12 exclusively, whereas 16.10 uses libpng16, and the opposite version cannot be found on their own apt sources.

Solutions would be to produce two artefacts, or potentially include our own library in Citra releases (what is it used for, anyway?)

**Comment by yuriks** [comment]
It's used for texture dumping, I think? (But I'm not sure how you even enable that.) We can probably disable that, but given that our CI is meant to check if we can actually compile our code that would be a downgrade in usefulness because it wouldn't cover that codepath. Updating the Travis CI container version is a pain (because it tends to randomly break stuff) and would break compatibility with older distros.

Basically, you'll just have to compile your own for now.
